web/src/message_list_tooltips.ts

```diff
@@ -262,23 +261,11 @@ export function initialize(): void {
 
     message_list_tooltip(".code_external_link");
 
-    message_list_tooltip(".change_visibility_policy > i", {
-        delay: LONG_HOVER_DELAY,
-        onShow(instance) {
-            const $elem = $(instance.reference);
-            const current_visibility_policy_str = $elem.attr("data-tippy-content");
-            instance.setContent(
-                parse_html(
-                    render_change_visibility_policy_button_tooltip({current_visibility_policy_str}),
-                ),
-            );
-        },
-        onHidden(instance) {
-            instance.destroy();
-        },
+    message_list_tooltip("#message_feed_container .change_visibility_policy > i", {
+        ...topic_visibility_tooltip_prop,
     });
 
-    message_list_tooltip(".recipient_bar_icon", {
+    message_list_tooltip("#message_feed_container .recipient_bar_icon", {
```

Comment:
It looks like the `recipient_bar_icon` tooltip change could have been its own prep refactor commit.